cli/src/main/java/com/devonfw/tools/ide/tool/LocalToolCommandlet.java

```diff
@@ -352,10 +334,7 @@ protected VersionIdentifier getInstalledVersion(Path toolPath) {
   @Override
   public String getInstalledEdition() {
 
-    if (this.context.getSoftwarePath() == null) {
-      return "";
-    }
-    return getInstalledEdition(this.context.getSoftwarePath().resolve(this.tool));
```

Comment:
this was a bug: For GraalVM we were saying the tool is not installed even if it was.